src/base/bittorrent/bandwidthscheduler.cpp

```diff
@@ -75,20 +75,21 @@ bool BandwidthScheduler::isTimeForAlternative() const
     {
         switch (schedulerDays)
         {
-        case EVERY_DAY:
+        case Scheduler::Days::EveryDay:
             alternative = !alternative;
             break;
-        case WEEK_ENDS:
+        case Scheduler::Days::Weekend:
             if ((day == 6) || (day == 7))
                 alternative = !alternative;
             break;
-        case WEEK_DAYS:
+        case Scheduler::Days::Weekday:
             if ((day != 6) && (day != 7))
                 alternative = !alternative;
             break;
         default:
-            if (day == (schedulerDays - 2))
+            if (day == (static_cast<int>(schedulerDays) - 2))
```

Comment:
Maybe change `Scheduler::Days` values to follow day numbers (i.e. `Monday = 1` etc.)? Extra items can have zero and negative values.

Reply:
That would break the migration code... I've pushed an update to resolve the deficiency.

Reply:
To be honest, I meant simplifying the code...

Reply:
>To be honest, I meant simplifying the code...

I would appreciate if you give some examples for that and it would not break easily on future updates...

Reply:
Sorry for the delay. I just meant that if the numerical values of the enumeration would coincide with the numbers of the days, then the code in this place would look clearer. On the other hand, it can be improved if you initially convert the day number into an enumeration type: `const auto day = static_cast<Scheduler::Days>(...`.